iron-selectable.html

```diff
@@ -241,6 +242,12 @@
       this._setItems(nodes);
     },
 
+    _updateAttrForSelected: function() {
+      if (this._shouldUpdateSelection) {
+        this.selected = this._indexToValue(this.indexOf(this.selectedItem));        
```

Comment:
I though about this solution to keep only one observer, but, yeah, separation of concerns is indeed cleaner